.github/dependabot.yml

```diff
@@ -19,7 +19,7 @@ updates:
   reviewers: [MaterializeInc/dependency-czars]
   labels: [A-dependencies]
 - package-ecosystem: docker
-  directory: /misc/ubuntu-base-image
+  directory: /misc/ubuntu-base
```

Comment:
I don't know what this change is, so maybe it's stray? Everything else looks fine to me.

Reply:
That's right!